packages/php/composer.json

```diff
@@ -14,19 +14,21 @@
     "readme"
   ],
   "require": {
-    "php": "^8.0 | ^8.1 | ^8.2",
-    "illuminate/http": "^9.0 | ^10.0",
-    "illuminate/support": "^9.0 | ^10.0",
-    "ramsey/uuid": "^3.7 | ^4.0",
-    "composer/composer": "^2.0",
-    "guzzlehttp/guzzle": "^7.0",
-    "composer-runtime-api": "^2.2"
+    "php": "^8.1 | ^8.2 | ^8.3",
+    "illuminate/http": "^10.0 | ^11.0",
+    "illuminate/support": "^10.0 | ^11.0",
+    "ramsey/uuid": "^4.7",
+    "composer/composer": "^2.7",
+    "guzzlehttp/guzzle": "^7.9",
+    "composer-runtime-api": "^2.2",
+    "phpoption/phpoption": "^1.9",
+    "vlucas/phpdotenv": "^5.6"
```

Comment:
what are these?

Reply:
There were some errors on the new versions:
```
Uncaught Error: Call to undefined method Dotenv\Repository\RepositoryBuilder::create
Error: Class "PhpOption\Option" not found
```
And this installed packages seems to fix them.